compiler/crates/relay-compiler/src/remote_persister.rs

```diff
@@ -20,7 +35,10 @@ impl OperationPersister for RemotePersister {
     ) -> Result<String, PersistError> {
         let params = &persist_config.params;
         let url = &persist_config.url;
-        persist(&operation_text, url, params).await
+        let permit = &self.semaphore.acquire().await.unwrap();
```

Comment:
@robrichard can we make `semaphore` part of the `persist_config` (but ignore it in `serde`)?

Then, this line became `&persist_config.semaphore.acquire().await.unwrap()`?

Reply:
@alunyov updated. `semaphore` is now part of the `persist_config`.

Reply:
Thanks! This looks great! One last minor request:

Can you add this option to the docs here:
https://github.com/facebook/relay/blob/main/packages/relay-compiler/README.md

Reply:
@alunyov docs are updated. Thanks!